glusterd2/volume/struct.go

```diff
@@ -97,6 +97,7 @@ type Volinfo struct {
 	Subvols   []Subvol
 	Auth      VolAuth // TODO: should not be returned to client
 	GraphMap  map[string]string
+	HealFlag  bool
```

Comment:
Name can be changed to `HealEnabled` or `SelfHealEnabled`

Reply:
Resolved